docs/public/installation/zonemaster-engine.md

```diff
@@ -168,11 +168,11 @@ installation above but without the `--notest` and with the `--verbose` option.
 Installation will take longer time.
 
 
-## Global cache (experimental feature)
+## Global cache
 
-Global cache is an experimental feature that can be enabled in Zonemaster-Engine
-and that can increase the performance when many tests are run within a short time
-frame. See [global cache configuration].
+Global cache is a feature that can be enabled in Zonemaster-Engine and consists
+in a shared, persistent cache. It can increase the performance when many tests
+are run within a short time frame. See [global cache configuration].
```

Comment:
I think it could be much shorter in the Engine installation instruction, just as in the Backend installation instruction. But if it should be more detailed it should be so in both places. Or actually in all three places, because there is text in the CLI installation instructions too https://github.com/zonemaster/zonemaster/blob/develop/docs/public/installation/zonemaster-cli.md#global-cache

Reply:
I think that shorter text is better. No need for longer text when there a separate document.